x-pack/auditbeat/module/system/host/host.go

```diff
@@ -311,16 +315,37 @@ func getHost() (*Host, error) {
 }
 
 func hostEvent(host *Host, eventType string, action eventAction) mb.Event {
-	return mb.Event{
+	hostFields := host.toMapStr()
+
+	event := mb.Event{
 		RootFields: common.MapStr{
 			"event": common.MapStr{
 				"kind":   eventType,
 				"action": action.String(),
 			},
 			"message": hostMessage(host, action),
 		},
-		MetricSetFields: host.toMapStr(),
-	}
+		MetricSetFields: hostFields,
```

Comment:
If it weren't a breaking change I'd suggest not duplicating values between `host.*` and `system.host.*`.

Reply:
Agreed